spec/lib/process_settings/target_spec.rb

```diff
@@ -334,4 +334,12 @@
       end
     end
   end
+
+  describe "class methods" do
+    describe "true_target" do
+      it "uses the TRUE_JSON_DOC" do
+        expect(described_class.true_target.json_doc).to eq(described_class::TRUE_JSON_DOC)
+      end
```

Comment:
```suggestion
      subject { described_class.true_target.json_doc }
      it { should eq({}) }
```

In the suggestion I switched away from using `described_class::TRUE_JSON_DOC` to avoid using the code under test to test itself.  I think here it's important to deliberately assert that the true state is empty hash so that accidental changes to the constant don't go without a double check.